apps/src/templates/sectionProgress/progressTables/ProgressTableTextCells.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const styles = {
+  text: {
+    display: 'flex',
```

Comment:
This seems like the type of thing we should be handling with class names instead of imports. I wonder if we should think about moving those reused style blocks into classes in `progressTableStyles` instead

Reply:
i'm skeptical of moving more into CSS classes since it sounds like we're not headed in that direction as an org. i used CSS for the table layout stuff for performance reasons and to separate table layout styling from component-specific styling to help keep the latter clearer in the components, but i was considering that a very intentional exception from our org norm.

as soon as we finish the react upgrade we can start using `emotion` and developing new patterns around it. until then, i'm of the opinion that we should continue to follow our existing norms rather than moving in an intermediate direction.

if we want to move toward more shared styles and style helpers, ideally we'd do that in a broader, more intentional way than adding them to `progressStyles`, which is why i haven't been pushing for more of this kind of thing yet. but i just wanted to start playing around with it to see how it feels. but like i said, this is a NIT, feel free to ignore if that doesn't interest you.